rest/notification/instance-delete-examples/meta.json

```diff
@@ -1,4 +1,4 @@
 {
-  "title": "instance-delete-examples",
+  "title": "Delete notification",
```

Comment:
Lets use title case in the titles, I'll mark all the errors in the PR so you can see them gone after the next commit